docs/sources/alerting/monitor/_index.md

```diff
@@ -1,27 +1,166 @@
 ---
+aliases:
+  - alerting/meta-monitoring/
```

Comment:
I've taken to the convention of adding a comment a the end of each line that includes the absolute path that we are redirecting. It makes it a lot easier to test the redirects and make sure that they get updated when we move files around.

Reply:
not sure what you mean?

Reply:
An example is in https://github.com/grafana/grafana/pull/74673/files#diff-3ed03901c3fa92bdbec5da7e20588f4dc3bd1e4e22ead03d84f0fe0a68189b82R3-R9

Reply:
ok cool done!